bitcoin/examples/create-p2wpkh-address.rs

```diff
@@ -0,0 +1,27 @@
+extern crate bitcoin;
+
+use bitcoin::{Address, CompressedPublicKey, PrivateKey, Network};
+use bitcoin::secp256k1::{rand, Secp256k1};
+
+// Generate a P2WPKH (pay-to-witness-public-key-hash) address and print it
+// along with the associated private key needed to transact.
+
+fn main() {
+    // Create a new secp256k1 elliptic curve.
```

Comment:
This comment does not mean anything, right? How does one "create a new curve".

Reply:
we are using a secp256k1 elliptic curve library here to generate new points.  I'll update the comment though if it helps.